tests/run_all_tests.sh

```diff
@@ -1,5 +1,22 @@
 #!/bin/bash
 
+# Download TFDS metadata to flax/.tdfs/metadata directory. 
+# This allows the tests to specify the `data_dir` when using tfds.testing.mock_data().
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)"
+DIR="$( dirname "${DIR}" )"
+
+if [ -d "${DIR}/.tfds/metadata" ]; then 
+  echo 'Exists'; 
+else 
+  echo 'TFDS metadata doesnt exist. Downloading...';
+  # subversion checkout to the `trunk` branch which corresponds to `tree/master`.
+  # To download from branch `foo`, replace `trunk` with `branches/foo`.
+  svn checkout \
```

Comment:
Add a comment before saying, e.g. "We use subversion for faster downloads of only the metadata subdirectory with no history. If this script fails for you because you don't have subversion installed, please let us know by opening a GitHub issue"

Reply:
Done.